data/colors_v2/themes/dark_high_contrast.ts

```diff
@@ -125,8 +125,13 @@ const exceptions = {
       disabledBorder: alpha(get('scale.green.1'), 0.4),
       icon: get('scale.gray.9'),
       counterBg: alpha(get('scale.black'), 0.15)
+    },
+  },
+  globalNav: {
+      inputBorder: get('scale.gray.5'),
```

Comment:
Should this map to `border.default` instead?

Reply:
Yes, I was thinking about yesterday evening as well. I guess it’s a better way to fix this.